frontend/src/metabase/query_builder/containers/use-create-question.ts

```diff
@@ -11,7 +11,7 @@ import type { DashboardTabId } from "metabase-types/api";
 
 import { updateUrl } from "../actions/url";
 
-type OnCreateOptions = { dashboardTabId?: DashboardTabId | undefined };
+export type OnCreateOptions = { dashboardTabId?: DashboardTabId | undefined };
```

Comment:
Used in `enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx`